chef/cookbooks/provisioner/recipes/update_nodes.rb

```diff
@@ -136,7 +136,7 @@
           hostname mnode.name
           macaddress mac_list[i]
           if mnode.macaddress == mac_list[i]
-            ipaddress admin_data_net.address
+            ipaddress admin_data_net.address unless admin_data_net.nil?
```

Comment:
Just to confirm: when that condition is not met, we don't put an IP address in the config file for the node; does it still get an IP address from the right pool (dhcp) of the admin network? (if you reboot, for instance)